Gemfile.lock

```diff
@@ -781,7 +781,7 @@ DEPENDENCIES
   web-console (~> 3.5)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.1p83
```

Comment:
Ruby version in `Gemfile.lock` must always be equal to [.ruby-version](https://github.com/OpenSourcePolitics/decidim-module-question_captcha/blob/add_compatibility_signup/.ruby-version) 

Can you revert this change pls ?